services/manage/app/routes/manage/reporting.js

```diff
@@ -0,0 +1,28 @@
+import Route from '@ember/routing/route';
+import moment from 'moment';
+
+import query from '@base-cms/parcel-plug-manage/gql/queries/reporting/list';
+import RouteObservableMixin from '../../mixins/route-observable-mixin';
+
+export default Route.extend(RouteObservableMixin, {
+  /**
+   *
+   * @param {object} params
+   */
+  async model() {
+    const start = parseInt(moment().startOf('week').format('x'));
```

Comment:
To make more clear (and to remove the need to `parseInt`) do the following
`const start =  moment().startOf('week').valueOf()`